test/Microsoft.IIS.Administration.Tests/Sites.cs

```diff
@@ -557,7 +558,7 @@ private void WaitForStatus(HttpClient client, ref JObject site)
         private static JObject GetCertificate(HttpClient client)
         {
             string result;
-            if (!client.Get(CertificatesUrl + "?intended_purpose=server authentication", out result)) {
+            if (!client.Get(CertificatesUrl + $"?intended_purpose={OIDServerAuth}", out result)) {
```

Comment:
We need a reference document with the number string so that users can find the correct string.

Reply:
Note:
1. This is publicly documented
2. You can also use "Server Authentication" to query. It would also work